mllib/src/main/scala/org/apache/spark/ml/feature/CountVectorizer.scala

```diff
@@ -42,7 +42,8 @@ private[feature] trait CountVectorizerParams extends Params with HasInputCol wit
    * vocabSize terms ordered by term frequency across the corpus.
    *
    * Default: 2^18^
-   * @group param
+    *
+    * @group param
```

Comment:
Something wrong with my editor. I didn't change this line and even I remove the space, it still shows there. I change to VI now. Thanks for pointing it out.